azext_iot/operations/_mqtt.py

```diff
@@ -16,8 +16,8 @@ class mqtt_client(object):
     def __init__(self, target, device_conn_string, device_id, method_response_code=None, method_response_payload=None):
         self.device_id = device_id
         self.target = target
-        self.device_client = mqtt_device_client.create_from_connection_string(device_conn_string)
-        self.device_client.connect()
+        # The client automatically connects when we send/receive a message or method invocation
+        self.device_client = mqtt_device_client.create_from_connection_string(device_conn_string, websockets=True)
```

Comment:
So just curious - what effect does this have (automatic  `connect()`) on the client initialization overhead? Is it about the same, or does this allow us to send single messages more quickly?

Reply:
The message sending and operations take half the time now. The same set of messaging integration tests which too about ~20 minutes earlier now finish in ~10 minutes. So major improvement I would say.